pkg/asset/tls/tls.go

```diff
@@ -24,6 +24,9 @@ const (
 	// ValidityOneDay sets the validity of a cert to 24 hours.
 	ValidityOneDay = time.Hour * 24
 
+	// ValidityOneMinute sets the validity of a cert to 1 month.
+	ValidityOneMonth = ValidityOneDay * 30
```

Comment:
Same comment as here: https://github.com/openshift/installer/pull/4308#discussion_r530360087

This is a no-go situation. Initial certificates are deemed to be only 24hrs, after which they rotate to the expected 30 days validity.